frontend/package.json

```diff
@@ -10,6 +10,7 @@
     "carbon-components-react": "^6.6.3",
     "carbon-icons": "^7.0.7",
     "date-fns": "^1.29.0",
+    "detect-browser": "^4.8.0",
```

Comment:
There's another package.json file you need to add this too. The level above frontend. It's for heroku

Reply:
Once you've done that it's ready to go